### PR TITLE
Iteration2/init data js

### PR DESCRIPTION
**Pull request by jonthanny**
In this PR (group programmed):

- We added an instance of the Outfit class to main.js
- Defined global variables for outfits and added the corresponding classes to HTML
- Added js class tags to left column buttons
- added event listeners for every category on the left column
- added an initial function architecture for the left side buttons

### CODE REVIEW

**Comment by ryanbahan** [comment]
nice work. hopefully we can find a way to reduce our event listener's function length down the road.

File: index.html [hunk 1]
```diff
@@ -11,42 +11,41 @@ <h1 class="page-title">BUILD-A-BEAR!</h1>
...
       <section class="outfit-options flex-column">
-        <section class="hat-options outfit-card">
+        <section class="hat-options hat-options-js outfit-card">
           <h2>Hats</h2>
           <div class="button-container">
-            <button>Top Hat</button>
-            <button>Sun Hat</button>
-            <button>Bow</button>
-            <button>Crown</button>
+            <button class="top-hat-js">Top Hat</button>
```
Comment (Karlfunhouse) [inline index.html]: sooo mannny classssesssss!  Nice organization up front.  This will be so helpful in the future.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+var newOutfit = new Outfit(null, null, null, null);
```
Comment (Karlfunhouse) [inline main.js]: The beginning of something beautiful.  JSquad Goals accomplished!

File: main.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+var accessoriesOptions = document.querySelector('.accessories-options-js');
+var backgroundOptions = document.querySelector('.background-options-js');
+console.log(newOutfit);
+
+hatOptions.addEventListener('click', selectHat);
+clothesOptions.addEventListener('click', selectClothes);
+accessoriesOptions.addEventListener('click', selectAccessories);
+backgroundOptions.addEventListener('click', selectBackground);
+
+function selectHat() {
```
Comment (Karlfunhouse) [inline main.js]: I wonder how we will be able to keep these under 10 lines...tbd.

**Pull request merged**